File: JAVA/Daily Flash Codes/Week1/Day3_Jan15/code_java/program2.java
/*
 Program 2: Write a Program to take a integer raging between 0 to 6 and print
corresponding weekday
Input: 2
Output: Wednesday.
 */


import java.io.*;

class Program {
	public static void main(String args[]) throws IOException {
		
		int  i;

	do {
		System.out.println("Enter Integer value(0-6): ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in) );
		i = Integer.parseInt(br.readLine());	//to read integerr

		if(i < 0 || i > 6)
			System.out.println("Wrong input !! Try again ");
	}while(i <0 || i > 6);

	switch(i) {
		case 0:
			System.out.println("Monday");
		        break;	
		case 1:
			System.out.println("Tuesday");
		        break;	

		case 2:
			System.out.println("Wednesday");
		        break;	
		case 3:
			System.out.println("Thursday");
		        break;	
		case 4:
			System.out.println("Friday");
		        break;	
		case 5:
			System.out.println("saturday");
		        break;	
		case 6:
			System.out.println("Sunday");
		        break;	
				
	}



	}
}

File: PYTHON/Daily Flash/week1/Day3_Jan15/code_python/program2.py
'''
Program 2: Write a Program to take a integer raging between 0 to 6 and print
corresponding weekday
Input: 2
Output: Wednesday.
'''


#take integer value

while True:
    var = int( input("Enter interger value(0-6): "))
    if(var < 0 or var > 6):
        print("Wrong input value!! Enter Again")
        continue
    break
    

if(var == 0):
    print("Monday")
elif( var == 1):
    print("Tuesday")
elif( var == 2):
    print("Wednesday")
elif( var == 3):
    print("Thursday")
elif( var == 4):
    print("Friday")
elif( var == 5):
    print("Saturday")
else:
    print("Sunday")





